### ISSUE TITLE
Articles for Issue #3

### ISSUE THREAD

**Issue by rebeccarubenstein** [midnightbreakfast/web#129]
This will be our official rendezvous point for setting up articles for Issue 3. 

@nevanscott Taylor is going to handle setting up his own articles in GH for this issue. Could you set us up for Issue 3, so that Taylor and I can add the finished pieces to GH as Lauren sends them to us? (This will expedite the process, I think.) Could you also provide us with a link to where the pieces will live and also specific set-up/editorial instructions for Taylor (see: closed issue ticket #107 for reference)? 

@ftpavlik If you have any questions at all, use this thread as a home base.

**Comment by nevanscott** [comment]
I got things started with one article, still with some placeholder content:

https://github.com/midnightbreakfast/web/tree/magazine/_src/magazine/_posts/issue003

But I also attempted to start documenting this like I did for Small Plates:

https://github.com/midnightbreakfast/web/wiki/Magazine-Issues

Let me know if either of you have any questions.

**Comment by rebeccarubenstein** [comment]
Thanks so much!

@nevanscott, I do have one quick question. Here's a screenshot, but since you've changed the formatting on this slightly, what should we put in the 'url' field beneath 'author'? 

![set-up for mb issue articles](https://cloud.githubusercontent.com/assets/4043562/4021042/6768c56c-2ade-11e4-969a-8fcc2fc504e0.png)

**Comment by nevanscott** [comment]
It should just be the author's website or Twitter. That gets used for the link that their name goes to on the Contributors page.

**Comment by rebeccarubenstein** [comment]
What if their name doesn't have a link? There are at least two bios this time around that don't.

**Comment by rebeccarubenstein** [comment]
You know what, never mind; I'll just sort it out so the names link.

**Comment by nevanscott** [comment]
For now if they really have no URL just leave it blank.

**Comment by nevanscott** [comment]
Or rather, remove that line.

**Comment by rebeccarubenstein** [comment]
All the names link to Twitter accounts for this issue -- it's not a problem anymore.

**Comment by ftpavlik** [comment]
Sorry I'm late to the party. Should I add the twitter accounts to the URL field or is this something you're handling now?

Thanks!

**Comment by rebeccarubenstein** [comment]
@ftpavlik Please add the Twitter URLs yourself. The names in the bios should still link to them, but I guess Nevan has it set up where this somehow auto-generates the Contributors page, and the names will link to Twitter (or personal websites) so long as we fill in the URL field properly? Also, since there are three of us on this thread, if you could @ mention whomever you're addressing, that'd be great.

**Comment by ftpavlik** [comment]
@rebeccarubenstein Thanks. I'll add twitter links there for now.

**Comment by rebeccarubenstein** [comment]
Reopening this until we're done adding all the articles.

**Comment by ftpavlik** [comment]
@rebeccarubenstein My two have been added.

**Comment by rebeccarubenstein** [comment]
@ftpavlik Please open separate issue tickets to let Nevan know when he can/should add art to your pieces. See #133 for example of how you can title them.

I'm still waiting on Molly to get back to me about some copyedits, so I'm leaving this open until I've had a chance to add her story to the system.

Also, let's keep this issue ticket open until we've had a chance to proofread our pieces on the preview site.

@nevanscott Please use this space to let us know when the articles have been posted to the preview site.

**Comment by nevanscott** [comment]
I'd prefer to open another issue for that. I won't be able to update the preview site until I get the artwork in anyway, which will also be a separate issue...

**Comment by rebeccarubenstein** [comment]
@nevanscott All right, I'll just leave this open until I've got Molly's piece loaded in.

**Comment by nevanscott** [comment]
@ftpavlik Also wanted to say that in general we’ve been using Markdown instead of straight HTML, so links would look like:

``` md
[link text](http://example.com/)
```

instead of 

``` html
<a href=“http://example.com/“>link text</a>
```

**Comment by rebeccarubenstein** [comment]
@ftpavlik Could you also go through your pieces and make sure all em dashes have spaces between them and the text? So, sentences would look like:

"This is a good — cheap, but good — piano."

We've been implementing this stylistic choice for all of our issues, but I think I forgot to communicate this with Lauren, so my pieces came back with all spacing removed.

Once you've done this (and the Markdown thing with links), and your pieces are ready to go, feel free to confirm and then close this ticket. Molly's piece is uploaded and ready, as are all my other pieces.

**Comment by ftpavlik** [comment]
@rebeccarubenstein @nevanscott All set with Markdown URLs and em-dash spaces. Are we moving ahead to preview?

**Comment by rebeccarubenstein** [comment]
@ftpavlik If you're all good with your pieces, please close this issue ticket and open new ones so that @nevanscott knows to implement art (see #138 for example).

**Comment by nevanscott** [comment]
Just thought I'd capture this here: http://preview.midnightbreakfast.com should be updated right now.

**Comment by ftpavlik** [comment]
@nevanscott It's much to late for this issue, but going forward could you please send me whatever rough draft you get of the art for my pieces, just in case there's something I think the author would want changed? The cake in "Iced Cake Frosting," for example, is described as white with white frosting.

**Comment by nevanscott** [comment]
Shoot, I'm sorry I missed that...
